src/components/pages/FirstPage.jsx

```diff
@@ -8,25 +9,15 @@ export const FirstPage = () => {
   const [editTaskIndex, setEditTaskIndex] = useState(null);
 
 useEffect(() => {
-  const fetchTasks = async () => {
-    try {
-      const response = await fetch("/todos");
-      const data = await response.lson();
-      setTaskList(data);
-    } catch(error) {
-        console.log("Ошибка при загрузке задач:", error);
-    }
-  };
-
-  fetchTasks();
+  FetchTasks(setTaskList);
```

Comment:
First letter should be lowercase for functions (fetchTasks), only class names and some other cases are with first uppercase letter